frontend/packages/system/src/Notifications/Pages/Base/NotificationDetailPanel.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  DeleteIcon,
+  DetailPanelPortal,
+  PanelLabel,
+  PanelRow,
+} from '@common/ui';
+import {
+  BufferedTextInput,
+  DetailPanelSection,
+  IconButton,
+} from '@common/components';
+import { KeyedParams } from '@common/utils';
+import { useTranslation } from '@common/intl';
+
+export interface ParamsPanelProps {
+  requiredParams: string[];
+  params: KeyedParams;
+  onUpdateParams: (key: string, value: string) => void;
+  onDeleteParam: (key: string) => void;
+}
+
+export const NotificationDetailPanel = ({
+  requiredParams,
+  params,
+  onUpdateParams,
+  onDeleteParam,
+}: ParamsPanelProps) => {
+  const t = useTranslation('system');
+
+  const allParams = [...new Set(requiredParams.concat(Object.keys(params)))];
+
+  const paramEditor = (
+    <DetailPanelSection title={t('label.parameters')}>
+      {allParams.map(param => {
+        return (
+          <>
+            <PanelRow key={`param-label-row-${param}`}>
+              <PanelLabel key={`param-label-${param}`}>{param}</PanelLabel>
+            </PanelRow>
+            <PanelRow key={`param-value-row-${param}`}>
+              <BufferedTextInput
+                key={`param-value-${param}`}
+                InputProps={{
+                  sx: {
+                    backgroundColor: 'white',
```

Comment:
so that the colour is picked from the theme, rather than a hard coded `white` string
```suggestion
                    backgroundColor: 'background.white',
```

Reply:
Cool thanks! Didn't realise the difference